packages/compass-indexes/src/components/indexes-table/indexes-table.tsx

```diff
@@ -178,6 +175,14 @@ export const IndexesTable: React.FunctionComponent<IndexesTableProps> = ({
                 )}
               </Cell>
             )}
+            <Row>
+              <Cell
+                className={cx(nestedRowCellStyles, cellStyles)}
+                colSpan={canDeleteIndex ? 6 : 5}
+              >
+                <KeyList keys={index.fields.serialize()} />
+              </Cell>
+            </Row>
           </Row>
```

Comment:
Small thing, bringing it up since we're updating some other parts of this table, feel free to do nothing. I found it a bit hard to know exactly which row I was deleting when hovering over the delete button. Should we do something like alternating the color of rows, or having a row hover background color to make it more apparent which row is being highlighted to delete? 
<img width="1141" alt="Screen Shot 2022-12-15 at 5 13 09 PM" src="https://user-images.githubusercontent.com/1791149/207979554-6cebfd5f-5c00-4324-914b-95f95b5c3231.png">

Reply:
Yeah that would be nice, but we may need some design input for that though